datastore/db/bytebase/schema_nachet_0.0.11.sql

```diff
@@ -0,0 +1,196 @@
+
+    CREATE TABLE "nachet_0.0.11"."object" (
+        "id" uuid NOT NULL DEFAULT uuid_.uuid_generate_v4() PRIMARY KEY,
+        "box_metadata" json NOT NULL,
+        "inference_id" uuid NOT NULL REFERENCES "nachet_0.0.11".inference(id),
+        "type_id" integer NOT NULL REFERENCES "nachet_0.0.11".object_type(id),
+        "verified_id" uuid ,
+        "valid" boolean NOT NULL DEFAULT true,
+        "top_id" uuid,
+        "upload_date" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        "updated_at" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    );
+    
+    
+    CREATE TABLE "nachet_0.0.11"."pipeline" (
+        "id" uuid NOT NULL DEFAULT uuid_.uuid_generate_v4() PRIMARY KEY,
+        "name" text NOT NULL,
+        "active" boolean NOT NULL DEFAULT false,
+        "is_default" boolean not null default false
+    );
+   
+    CREATE TRIGGER "pipeline_default_trigger" BEFORE insert OR UPDATE ON "nachet_0.0.11"."pipeline"
+    FOR EACH ROW EXECUTE FUNCTION "nachet_0.0.11".pipeline_default_trigger();
+
+    CREATE FUNCTION "nachet_0.0.11".pipeline_default_trigger() RETURNS TRIGGER AS $$
+    BEGIN
+        IF NEW.is_default THEN
+            UPDATE "nachet_0.0.11".pipeline SET is_default=false WHERE is_default=true;
+        END IF;
+        RETURN NEW;
+    END;
+   $$language plpgsql;
+
+    CREATE TABLE "nachet_0.0.11"."pipeline_default" (
+        "id" uuid NOT NULL DEFAULT uuid_.uuid_generate_v4() PRIMARY KEY,
+        "pipeline_id" uuid NOT NULL REFERENCES "nachet_0.0.11".pipeline(id),
+        "user_id" uuid NOT NULL REFERENCES "nachet_0.0.11".users(id)
+    );
+
+    CREATE UNIQUE INDEX "nachet_0.0.10_pipeline_default" ON "nachet_0.0.11".pipeline ("is_default") where is_default=true;
+    
+    CREATE TABLE "nachet_0.0.11"."pipeline_model" (
+        "id" uuid NOT NULL DEFAULT uuid_.uuid_generate_v4() PRIMARY KEY,
+        "pipeline_id" uuid NOT NULL REFERENCES "nachet_0.0.11".pipeline(id),
+        "model_id" uuid NOT NULL REFERENCES "nachet_0.0.11".model(id)
+    );
+    
+    CREATE TABLE "nachet_0.0.11"."seed_obj" (
+        "id" uuid NOT NULL DEFAULT uuid_.uuid_generate_v4() PRIMARY KEY,
+        "seed_id" uuid NOT NULL REFERENCES "nachet_0.0.11".seed(id),
+        "object_id" uuid NOT NULL REFERENCES "nachet_0.0.11".object(id)
+        "score" float NOT NULL
+    );  
+
+    CREATE OR REPLACE FUNCTION verified_inference() RETURNS TRIGGER LANGUAGE plpgsql AS $$
+        BEGIN
+            IF NEW.verified = true THEN
+                INSERT INTO picture_seed (seed_id, picture_id)
+                  SELECT 
+                    New.picture_id, 
+                    so.seed_id  
+                  FROM object obj 
+                    LEFT JOIN seed_obj so 
+                      ON so.object_id = obj.verified_id  
+                  WHERE obj.inference_id = NEW.id and obj.verified_id is not null;
+            END IF;
+        END;
```

Comment:
Got an error on the datastore when updating the verified column of an inference :
"control reached end of trigger procedure without RETURN"
I think the trigger should end with RETURN NULL or RETURN NEW